### ISSUE TITLE
German umlauts in Readme.md causes redmine to crash

### ISSUE THREAD

**Issue by Idleworks** [jbox-web/redmine_git_hosting#529]
When I enter some german umlauts to Readme.md in my main git repo (äöüß) redmine throws an error 500. My Readme.md is UTF-8 encoded.
Maybe this issue also exists with other special characters, like czech, turkish

Here's my production.log output
```
Started GET "/projects/na15-extensions/repository" for XXX.XXX.XXX.XXX at 2015-10-30 11:02:31 +0000
Processing by RepositoriesController#show as HTML
  Parameters: {"id"=>"na15-extensions"}
  Current user: idler (id=3)
  Rendered repositories/_navigation.html.erb (22.1ms)
  Rendered repositories/_breadcrumbs.html.erb (1.8ms)
  Rendered repositories/_dir_list_content.html.erb (109.1ms)
  Rendered repositories/_dir_list.html.erb (112.9ms)
  Rendered repositories/_revision_graph.html.erb (3.1ms)
  Rendered repositories/_revisions.html.erb (44.2ms)
  Rendered plugins/redmine_git_hosting/app/views/repositories/show.html.erb within layouts/base (862.1ms)
Completed 500 Internal Server Error in 957ms

ActionView::Template::Error ("\xC3" from ASCII-8BIT to UTF-8):
    47:                        :id => @project, :page => nil, :key => User.current.rss_key})) %>
    48:    <% end %>
    49: 
    50:    <%= call_hook(:view_repositories_show_bottom) %>
    51: 
    52:    <% other_formats_links do |f| %>
    53:     <%= f.link_to 'Atom',
  lib/redmine/hook.rb:61:in `block (2 levels) in call_hook'
  lib/redmine/hook.rb:61:in `each'
  lib/redmine/hook.rb:61:in `block in call_hook'
  lib/redmine/hook.rb:58:in `tap'
  lib/redmine/hook.rb:58:in `call_hook'
  lib/redmine/hook.rb:167:in `call_hook'
  app/controllers/repositories_controller.rb:125:in `show'
```

**Comment by Devpap7** [comment]
Thats intereseting. My umlauts in any readme.md are just mispreseted.

**Comment by n-rodriguez** [comment]
This should be fixed in next release (1.1.5) : https://github.com/jbox-web/redmine_git_hosting/commit/7cb9c3bdc8eff249177b714c06a6a544e308d0b2

**Comment by n-rodriguez** [comment]
Already fixed in v1.1.4 : https://github.com/jbox-web/redmine_git_hosting/pull/517
